pipelines/examples/namespaces/sandbox-canary/deployment-keys.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Secret
```

Comment:
this is junk in this example ... could be removed, just wanted to show that it is possible to add any resources to the namespace config dir